### ISSUE TITLE
Learning rate for training

### ISSUE THREAD

**Issue by datduong** [bio-ontology-research-group/deepgo#17]
Hi, would you be able to tell me the learning rate and batch size used for each MF, CC, BP ontology? I see that the default is 0.001 in the model that uses only amino sequence. Is 0.001 used for all experiments? Thanks.

**Comment by coolmaksat** [comment]
I tried different learning rates ranging from 0.0001 to 0.1, but If I'm not mistaken 0.001 was used in the
latest experiments.

**Comment by datduong** [comment]
Do you know if Keras optimizer has some special initialization for unknown parameters? I implemented your code in Pytorch. I am getting very similar AUC. However Fmax value is consistently lower than Keras (about 4%) lower. 

Thanks.

**Comment by coolmaksat** [comment]
I'm not sure. You can compare the default parameters https://keras.io/optimizers/
